build-system/amp.extern.js

```diff
@@ -30,6 +30,10 @@ window.context.experimentToggles;
 // Service Holder
 window.services;
 
+// Safeframe
+window.sf_ = {};
+window.sf_.cfg;
```

Comment:
I added typechecking when it is used, is that sufficient or do you want it here too?